src/lib/core/overlay/overlay-directives.ts

```diff
@@ -103,14 +105,19 @@ export class ConnectedOverlayDirective implements OnDestroy {
   constructor(
       private _overlay: Overlay,
       templateRef: TemplateRef<any>,
-      viewContainerRef: ViewContainerRef) {
+      viewContainerRef: ViewContainerRef,
+      @Optional() private _dir: Dir) {
     this._templatePortal = new TemplatePortal(templateRef, viewContainerRef);
   }
 
   get overlayRef(): OverlayRef {
     return this._overlayRef;
   }
 
+  get dir(): 'ltr' | 'rtl' {
```

Comment:
Type is `LayoutDirection` from `dir.ts`\

Do you need this getter at all, though, instead of just saying `this._dir.value`? It defaults to `ltr` in `Dir` itself.